frontend/src/__tests__/cypress/test-variables.yml.example

```diff
@@ -3,6 +3,10 @@ TEST_USER:
   AUTH_TYPE: foo-auth
   USERNAME: foo-user
   PASSWORD: foo-passwd
+TEST_USER_3:
```

Comment:
It would be good to define the other users, but as discussed on IM we can take that later.